Diagnostic/rsyslog5/omazurelinuxmds.conf

```diff
@@ -25,4 +25,4 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueFileName lad_mdsd_queue
 $ActionQueueDiscardSeverity 8
 local6.* :omazuremds:;fmtfile
-& ~
+& stop
```

Comment:
Thanks for confirming @mabicca . Please allow me to apply this change only to ver. 7 & 8, to reduce your inconvenience. We are planning to release a newer version of LAD soon, so this change should be included there for sure, together with the queue file name change.